dwave/system/utilities.py

```diff
@@ -89,3 +93,55 @@ def get(name, default=False):
     @classproperty
     def hss_solver_config_override(cls):
         return cls.get('hss_solver_config_override')
+
+
+def schedule_with_anneal_offset(
+        schedule_csv_filename: str,
```

Comment:
Oddly specific interface (downloading the schedule in xls format, convert it to csv, then feed it to this function, via file system).

This function should be decoupled from any CSV format specifics and file loading -- instead it should accept parsed `s, A, B, c` values, either as a numpy matrix, or 4 distinct vectors. Also, I'd call it something like `anneal_schedule_with_offset`.

For convenience, we can provide additional helper function that loads the CSV and then calls the interpolation function.

Reply:
[This commit ](https://github.com/dwavesystems/dwave-system/pull/568/commits/12c16eb3aef16cf1cb1fd44a9572b8ab6e1b6620) switches the code to accepting array-like inputs, either a 4D schedule or four 1D vectors. 
I'll add additional unittests for [this comment](https://github.com/dwavesystems/dwave-system/pull/568#discussion_r2068833235) separately.